visualizations/frontend/filters/v2/TDatePicker.vue

```diff
@@ -22,6 +22,9 @@
         <div class="flex items-center gap-1">
           <calendar-blank-outline-icon class="text-[16px]" />
           {{ relativePreset.label }}
+          <button @click="handleClear">
+            <close-icon />
+          </button>
```

Comment:
```suggestion
          <button @click="handleClear" >
            <close-icon title="clear date selection" />
          </button>
```

You can use whatever text makes sense, but without `title="clear date selection"` the button text for a screen reader would be `close icon`
I know we used a `visually-hidden` class in another component. This might replace that implementation.

Reply:
@sareg0 good call, thanks.